src/transformers/models/sam/convert_sam_original_to_pytorch.py

```diff
@@ -0,0 +1,199 @@
+# coding=utf-8
```

Comment:
This should be named `to_hf_format` as it's not converting anything to PyTorch (the model was already in PyTorch).